src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  width: $cell-size * 7 + $gap * 6;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    box-sizing: border-box;
+    background-color: $cell-color;
+    width: $cell-size;
+    height: $cell-size;
+    border: 1px solid black;
+
+    @for $i from 1 through $count-of-days {
+      &:nth-child(#{$i})::before {
+        content: '#{$i}';
+      }
+    }
+
+    &:hover {
+      cursor: pointer;
+      background-color: #ffbfcb;
+      transform: translate(0, -20px);
+      transition-duration: 0.5s;
+    }
+  }
+
+  @each $day, $margin in $week_days {
+    &--start-day-#{$day} &__day:first-child {
+      margin-left: if($margin == 0, 0, ($margin * $gap + $margin * $cell-size));
```

Comment:
Probably you can get rid of condition at all

Reply:
I don't really understand how to get rid of the condition? this piece of code allows me to change the distance to the cell depending on the day of the week

Reply:
I'm sure u can get rid of the condition here)
`0 * ... = 0`